woocommerce/woocommerce-android
# Update icon for subscription product types in bottom sheet

Closes: #10196 


### Description

Replaces money icon for product subscription type with `ic_gridicons_event_repeat` 

### Testing instructions

1. Log into a site eligible for product subscriptions
2. Click on "Add product" 
3. Check the subscription products have the new icon `ic_gridicons_event_repeat`

### Images/gif

![Screenshot 2023-11-17 at 09 19 43](https://github.com/woocommerce/woocommerce-android/assets/2663464/2b278862-80b9-4dd2-8c11-259e783df2a0)
